sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/TableProvider.java

```diff
@@ -28,6 +29,7 @@
  * <p>So there will be a provider to handle textfile(CSV) based tables, there is a provider to
  * handle MySQL based tables, a provider to handle Casandra based tables etc.
  */
+@Internal
```

Comment:
This is a public interface used to implement custom table providers. It can't be internal.

Reply:
Agree

Reply:
Done